src/components/message/index.tsx

```diff
@@ -420,7 +420,16 @@ export class Message extends React.Component<Properties, State> {
 
   renderLinkPreview() {
     const { preview, hidePreview, media, parentMessageText } = this.props;
-    if (!preview || hidePreview || media || parentMessageText) {
+    console.log('XXXpreview', preview);
+    if (
+      !preview?.title ||
```

Comment:
The preview will always have a url as it is extracted from the message that is typed and sent. Therefore I have added to checks for the other required fields.